### PR TITLE
Simple service for fetching the latest news on r/openSUSE

### PR DESCRIPTION
**Pull request by why-not-try-calmer**

### CODE REVIEW

File: defrag/modules/news.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Dict, List, NamedTuple, Optional
+
+from aiohttp.client import ClientSession
```
Comment (KaratekHD) [inline defrag/modules/news.py:19]: Could you add these two modules to the `requirements.txt`?

File: defrag/modules/news.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+from defusedxml.ElementTree import fromstring
+from collections import namedtuple
+import asyncio
+import aiohttp
+
+__MOD_NAME__ = "news"
```
Comment (KaratekHD) [inline defrag/modules/news.py:25]: I think I would call this one "Reddit", since we also have news.opensuse.org

**Comment by why-not-try-calmer** [comment]
Changed requirements, change module name.

File: defrag/modules/news.py (hunk 1 above)
Comment (why-not-try-calmer) [reply defrag/modules/news.py:18]: *[x]

File: defrag/modules/news.py (hunk 2 above)
Comment (why-not-try-calmer) [reply defrag/modules/news.py]: * [x]

**Comment by why-not-try-calmer** [comment]
Continued after rebasing onto the latest dev branch at https://github.com/KaratekHD/defrag/pull/23

**Pull request closed without merging**